apps/web/src/hooks/useGasPrice.ts

```diff
@@ -46,6 +46,10 @@ const isEtherscanResult = (data: any): data is EtherscanResult => {
   return 'FastGasPrice' in data && 'suggestBaseFee' in data
 }
 
+const stringToBigInt = (value: string): bigint => {
+  return BigInt(value.split('.')[0])
+}
```

Comment:
This could result in a too low gas price estimation as we are "rounding down" by cutting of the decimal part. Alternatively we could also round up. Wdyt?

Nit: The name of the function is not entirely accurate as it not only converts the value but also transforms it

Reply:
The point of this fn is to not convert to a number to avoid overflows but we could do something like `val === '0' ? '1' : val`.